package/Dockerfile.shipyard-dapper-base

```diff
@@ -79,6 +79,9 @@ RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/i
     find /go/bin /usr/local/libexec/docker/cli-plugins -type f -executable -newercc /proc \( -execdir upx ${UPX_LEVEL} {} \; -o -true \) && \
     go clean -cache -modcache
 
+# Link get-subctl script so it can be easily run inside a shell
+RUN ln -s $SCRIPTS_DIR/get-subctl.sh /root/.local/bin/subctl
```

Comment:
Because of the way `getsubctl.sh` (get.submariner.io) works, this can’t work — `getsubctl.sh` overwrites the target file instead of moving it into place. In this scenario, the `get-subctl.sh` script changes while it’s running, which results in

```
Installing subctl version devel
  OS detected:           linux
  Architecture detected: amd64
  Download URL:          https://github.com/submariner-io/subctl/releases/download/subctl-devel/subctl-devel-linux-amd64.tar.xz

Downloading...
subctl-devel-linux-amd64 has been installed as /root/.local/bin/subctl
This provides subctl version: devel
/opt/shipyard/scripts/get-subctl.sh: line 14: syntax error near unexpected token `$'\201\024\002''
```

_e.g._ in https://github.com/submariner-io/shipyard/runs/7605145306?check_suite_focus=true#step:7:1482

https://github.com/submariner-io/get.submariner.io/pull/30 will fix this.

Reply:
Hence why I added the line https://github.com/submariner-io/shipyard/pull/893/files#diff-b57b55d512de87ffb6a16ac5dbb0069de967a4bc2630f9ba5ca03f41355a05c0R10

But  seems that it didn't work for the consuming job for some reason, so I'm fine with getting the `get.submariner` fix first and then this one.

Reply:
Ah yes, that should have done the trick! But `command -v subctl` in the `subctl` repository will point to the `subctl` binary built in the repository, not `get-subctl.sh`-as-`subctl`.

Reply:
Yes that must be why it failed, well it should be working now.

Should I get rid of that line?